components/Header/Sidebar.tsx

```diff
@@ -18,41 +22,41 @@ type NavItem = {
   text?: string;
 };
 
-const navigationItems: NavigationItems[] = [
-  {
-    title: 'Newbies',
-    text: 'פעם ראשונה בקוד פתוח',
-    linkPath: '/newbies',
-  },
-  {
-    title: 'Members',
-    text: 'מי שכבר התנסה בקוד פתוח',
-    linkPath: '/members',
-  },
-  {
-    title: 'Maintainers',
-    text: 'בעלי פרויקטים שרוצים להצטרף',
-    linkPath: '/maintainers',
-  },
-  {
-    title: 'מי אנחנו',
-    linkPath: '/about',
-  },
-  {
-    title: 'הפרויקטים',
-    linkPath: '/projects',
-  },
-];
-
 interface SidebarProps {
   isOpen: boolean;
   toggleSidebar: () => void;
 }
 
 const Sidebar: React.FC<SidebarProps> = ({ isOpen, toggleSidebar }) => {
   const sidebarRef = useRef<HTMLDivElement | null>(null);
-
+  const localActive = useTypedLocale();
   const { theme } = useTheme();
+  const t = useTranslations('Components.sideBar');
+  const navigationItems: NavigationItems[] = [
```

Comment:
in this scope this array will be re declared on every render

Reply:
I know, but how can I use the "useTranslations" hook outside this scope?